src/interface/guide/components/DonutChart.tsx

```diff
@@ -0,0 +1,193 @@
+import styled from '@emotion/styled';
+import { formatPercentage, formatNumber } from 'common/format';
+import { useState } from 'react';
+
+export interface DonutSegment {
+  /** Unique identifier for the segment */
+  id: string | number;
+  /** Display label for the segment */
+  label: string;
+  /** Color for the segment */
+  color: string;
+  /** Numeric value for the segment */
+  value: number;
+}
+
+interface Props {
+  /** Array of segments to display */
+  segments: DonutSegment[];
+  /** Size of the donut chart in pixels */
+  size?: number;
+  /** Inner radius ratio (0-1, where 0.5 = half the radius) */
+  innerRadiusRatio?: number;
+  /** Whether to show the center text with total */
+  showCenterText?: boolean;
+  /** Custom center text override */
+  centerText?: string;
+}
+
+/**
+ * Base donut chart component that renders a simple donut visualization.
+ * For more complex use cases with spell tracking and legends, use DamageContribution.
+ *
+ * @param segments - Array of segments to display in the chart
+ * @param size - Size of the donut chart in pixels (default: 200)
+ * @param innerRadiusRatio - Inner radius ratio 0-1, where 0.5 = half radius (default: 0.6)
+ * @param showCenterText - Whether to show center text with total (default: true)
+ * @param centerText - Custom center text override
+ */
+export default function DonutChart({
```

Comment:
what is the purpose of having a new `DonutChart` (instead of using `parser/ui/DonutChart`)